app_dart/lib/src/request_handlers/scheduler/batch_backfiller_v2.dart

```diff
@@ -138,9 +137,9 @@ class BatchBackfillerV2 extends RequestHandler {
     }
     final List<Tuple<Target, FullTask, int>> filteredBackfill = <Tuple<Target, FullTask, int>>[];
     final List<Tuple<Target, FullTask, int>> highPriorityBackfill =
-        backfill.where((element) => element.third == LuciBuildService.kRerunPriority).toList();
+        backfill.where((element) => element.third == LuciBuildServiceV2.kRerunPriority).toList();
```

Comment:
The constants were duplicated when the service class was duplicated, but none of the code had updated to use the new copy of the constants. I checked that the values were all the same.